test_hidden_services still failing occasionally

```
=================================== FAILURES ===================================
_____________________________ test_hidden_services _____________________________
[gw5] linux -- Python 3.7.3 /usr/bin/python3

enable_ipv8 = None
proxy_factory = <test_full_session.test_tunnel_community.ProxyFactory object at 0x7f103aec6a90>
session = <tribler_core.session.Session object at 0x7f103aec69b0>
hidden_seeder_session = <tribler_core.session.Session object at 0x7f103b168390>
video_tdef = <tribler_core.modules.libtorrent.torrentdef.TorrentDef object at 0x7f103b1715f8>

    @pytest.mark.asyncio
    @pytest.mark.timeout(30)
    async def test_hidden_services(enable_ipv8, proxy_factory, session, hidden_seeder_session, video_tdef):
        """
        Test the hidden services overlay by constructing an end-to-end circuit and downloading a torrent over it
        """
        await load_tunnel_community_in_session(session, exitnode=False, start_lt=True)
    
        hidden_seeder_session.tunnel_community.build_tunnels(1)
    
        relays, exit_nodes = await create_nodes(proxy_factory, num_relays=3, num_exitnodes=2)
        await introduce_peers([session, hidden_seeder_session] + relays + exit_nodes)
        await deliver_messages(timeout=1)
    
        for ses in [session, hidden_seeder_session] + relays + exit_nodes:
>           assert len(ses.tunnel_community.get_peers()) == 6
E           assert 7 == 6
E             +7
E             -6

src/tribler-core/tribler_core/modules/tunnel/test_full_session/test_tunnel_community.py:233: AssertionError
---------------------------- Captured stdout setup -----------------------------
seeder: b'video.avi' DLSTATUS_EXIT_NODES 0
seeder: b'video.avi' DLSTATUS_HASHCHECKING 0.0
------------------------------ Captured log setup ------------------------------
DEBUG    asyncio:selector_events.py:53 Using selector: EpollSelector
INFO     Session:session.py:388 Session is using state directory: /tmp/pytest-of-jenkins/pytest-75/popen-gw5/test_hidden_services0/dot.Tribler
DEBUG    tribler_core.utilities.network_utils:network_utils.py:38 Got a working random port 24269
DEBUG    TriblerConfig:tribler_config.py:111 Get random port 24269 for [libtorrent~port]
DEBUG    tribler_core.utilities.network_utils:network_utils.py:38 Got a working random port 49162
DEBUG    TriblerConfig:tribler_config.py:111 Get random port 49162 for [libtorrent~anon_listen_port]
DEBUG    tribler_core.utilities.network_utils:network_utils.py:38 Got a working random port 48222
DEBUG    TriblerConfig:tribler_config.py:111 Get random port 48222 for [tunnel_community~socks5_listen_ports~0]
DEBUG    tribler_core.utilities.network_utils:network_utils.py:38 Got a working random port 50785
DEBUG    TriblerConfig:tribler_config.py:111 Get random port 50785 for [tunnel_community~socks5_listen_ports~1]
DEBUG    tribler_core.utilities.network_utils:network_utils.py:38 Got a working random port 42383
DEBUG    TriblerConfig:tribler_config.py:111 Get random port 42383 for [tunnel_community~socks5_listen_ports~2]
DEBUG    tribler_core.utilities.network_utils:network_utils.py:38 Got a working random port 6691
DEBUG    TriblerConfig:tribler_config.py:111 Get random port 6691 for [tunnel_community~socks5_listen_ports~3]
DEBUG    tribler_core.utilities.network_utils:network_utils.py:38 Got a working random port 39235
DEBUG    TriblerConfig:tribler_config.py:111 Get random port 39235 for [tunnel_community~socks5_listen_ports~4]
INFO     TrackerManager:tracker_manager.py:38 No tracker blacklist file found at /tmp/pytest-of-jenkins/pytest-75/popen-gw5/test_hidden_services0/dot.Tribler/tracker_blacklist.txt.
DEBUG    UDPEndpoint:endpoint.py:78 Listening at 7759
DEBUG    DiscoveryCommunity:community.py:76 Launching DiscoveryCommunity with prefix b'00027e313685c1912a141279f8248fc8db5899c5df5a'.
WARNING  Notifier:notifier.py:25 Called notification on a non-existing subject NTFY.TRIBLER_STARTED
DEBUG    DiscoveryCommunity:community.py:170 Bootstrapping DiscoveryCommunity, current peers 0
INFO     Session:session.py:388 Session is using state directory: /tmp/pytest-of-jenkins/pytest-75/popen-gw5/seeder1
DEBUG    tribler_core.utilities.network_utils:network_utils.py:38 Got a working random port 52141
DEBUG    TriblerConfig:tribler_config.py:111 Get random port 52141 for [libtorrent~port]
DEBUG    tribler_core.utilities.network_utils:network_utils.py:38 Got a working random port 46651
DEBUG    TriblerConfig:tribler_config.py:111 Get random port 46651 for [libtorrent~anon_listen_port]
DEBUG    tribler_core.utilities.network_utils:network_utils.py:38 Got a working random port 13656
DEBUG    TriblerConfig:tribler_config.py:111 Get random port 13656 for [tunnel_community~socks5_listen_ports~0]
DEBUG    tribler_core.utilities.network_utils:network_utils.py:38 Got a working random port 39197
DEBUG    TriblerConfig:tribler_config.py:111 Get random port 39197 for [tunnel_community~socks5_listen_ports~1]
DEBUG    tribler_core.utilities.network_utils:network_utils.py:38 Got a working random port 27601
DEBUG    TriblerConfig:tribler_config.py:111 Get random port 27601 for [tunnel_community~socks5_listen_ports~2]
DEBUG    tribler_core.utilities.network_utils:network_utils.py:38 Got a working random port 34290
DEBUG    TriblerConfig:tribler_config.py:111 Get random port 34290 for [tunnel_community~socks5_listen_ports~3]
DEBUG    tribler_core.utilities.network_utils:network_utils.py:38 Got a working random port 12321
DEBUG    TriblerConfig:tribler_config.py:111 Get random port 12321 for [tunnel_community~socks5_listen_ports~4]
INFO     TrackerManager:tracker_manager.py:38 No tracker blacklist file found at /tmp/pytest-of-jenkins/pytest-75/popen-gw5/seeder1/tracker_blacklist.txt.
DEBUG    tribler_core.utilities.network_utils:network_utils.py:38 Got a working random port 46793
DEBUG    TriblerConfig:tribler_config.py:111 Get random port 46793 for [ipv8~port]
DEBUG    UDPEndpoint:endpoint.py:78 Listening at 46793
DEBUG    DiscoveryCommunity:community.py:76 Launching DiscoveryCommunity with prefix b'00027e313685c1912a141279f8248fc8db5899c5df5a'.
WARNING  Notifier:notifier.py:25 Called notification on a non-existing subject NTFY.TRIBLER_STARTED
DEBUG    DiscoveryCommunity:community.py:170 Bootstrapping DiscoveryCommunity, current peers 0
DEBUG    TriblerTunnelCommunity:community.py:76 Launching TriblerTunnelCommunity with prefix b'0002caf481ffae562bcc0f206fd988ec0d246de0dc96'.
INFO     TriblerTunnelCommunity:community.py:140 Exit settings: BT=False, IPv8=False
ERROR    TriblerTunnelCommunity:triblertunnel_community.py:167 Could not retrieve backup exitnode cache, file does not exist!
INFO     DownloadManager:download_manager.py:240 could not load libtorrent state, got exception: FileNotFoundError(2, 'No such file or directory'). starting from scratch
DEBUG    DownloadManager:download_manager.py:255 Started libtorrent session for 0 hops on port 52141
DEBUG    DownloadManager:download_manager.py:720 Starting the download state callback with interval 1.000000
INFO     TriblerTunnelCommunity:community.py:179 Want 1 data circuits of length 1
INFO     TriblerTunnelCommunity:community.py:253 Creating a new circuit of length 1 (type: DATA)
INFO     TriblerTunnelCommunity:community.py:270 Could not create circuit, no available exit-nodes
INFO     TriblerTunnelCommunity:community.py:182 circuit creation of 1 circuits failed, no need to continue
DEBUG    DownloadManager:download_manager.py:492 Starting download: filename: None, torrent def: <tribler_core.modules.libtorrent.torrentdef.TorrentDef object at 0x7f103b1715f8>
DEBUG    Download:download.py:142 Setup: 9d5b2dbc52807325bfc28d688f2bb03f8d1e7667
DEBUG    Download:download.py:289 Saving download config to file /tmp/pytest-of-jenkins/pytest-75/popen-gw5/seeder1/dlcheckpoints/9d5b2dbc52807325bfc28d688f2bb03f8d1e7667.conf
DEBUG    DownloadManager:download_manager.py:387 Updating IP filter ['1.1.1.1']
DEBUG    DownloadManager:download_manager.py:255 Started libtorrent session for 1 hops on port 0
DEBUG    DownloadManager:download_manager.py:556 Adding handle 9d5b2dbc52807325bfc28d688f2bb03f8d1e7667
INFO     Socks5Connection:connection.py:90 Client has sent METHOD REQUEST
DEBUG    Socks5Connection:connection.py:113 Client has sent PROXY REQUEST
INFO     Socks5Connection:connection.py:174 Accepting UDP ASSOCIATE request to 127.0.0.1:50059
DEBUG    Download:download.py:183 Added torrent 9d5b2dbc52807325bfc28d688f2bb03f8d1e7667
DEBUG    Download:download.py:253 Setting IP filter for 9d5b2dbc52807325bfc28d688f2bb03f8d1e7667 to False
DEBUG    Download:download.py:289 Saving download config to file /tmp/pytest-of-jenkins/pytest-75/popen-gw5/seeder1/dlcheckpoints/9d5b2dbc52807325bfc28d688f2bb03f8d1e7667.conf
DEBUG    Download:download.py:253 Setting IP filter for 9d5b2dbc52807325bfc28d688f2bb03f8d1e7667 to False
DEBUG    Download:download.py:253 Setting IP filter for 9d5b2dbc52807325bfc28d688f2bb03f8d1e7667 to True
------------------------------ Captured log call -------------------------------
DEBUG    TriblerTunnelCommunity:community.py:76 Launching TriblerTunnelCommunity with prefix b'0002caf481ffae562bcc0f206fd988ec0d246de0dc96'.
INFO     TriblerTunnelCommunity:community.py:140 Exit settings: BT=False, IPv8=False
ERROR    TriblerTunnelCommunity:triblertunnel_community.py:167 Could not retrieve backup exitnode cache, file does not exist!
INFO     DownloadManager:download_manager.py:240 could not load libtorrent state, got exception: FileNotFoundError(2, 'No such file or directory'). starting from scratch
DEBUG    DownloadManager:download_manager.py:255 Started libtorrent session for 0 hops on port 24269
DEBUG    DownloadManager:download_manager.py:720 Starting the download state callback with interval 1.000000
INFO     TriblerTunnelCommunity:community.py:179 Want 1 data circuits of length 1
INFO     TriblerTunnelCommunity:community.py:253 Creating a new circuit of length 1 (type: DATA)
INFO     TriblerTunnelCommunity:community.py:270 Could not create circuit, no available exit-nodes
INFO     TriblerTunnelCommunity:community.py:182 circuit creation of 1 circuits failed, no need to continue
DEBUG    tribler_core.utilities.network_utils:network_utils.py:38 Got a working random port 45142
DEBUG    tribler_core.utilities.network_utils:network_utils.py:38 Got a working random port 43518
DEBUG    tribler_core.utilities.network_utils:network_utils.py:38 Got a working random port 18572
DEBUG    tribler_core.utilities.network_utils:network_utils.py:38 Got a working random port 42029
DEBUG    tribler_core.utilities.network_utils:network_utils.py:38 Got a working random port 37412
INFO     Session:session.py:388 Session is using state directory: /tmp/pytest-of-jenkins/pytest-75/popen-gw5/test_hidden_services0/dot.Tribler
DEBUG    tribler_core.utilities.network_utils:network_utils.py:38 Got a working random port 26481
DEBUG    TriblerConfig:tribler_config.py:111 Get random port 26481 for [libtorrent~port]
DEBUG    tribler_core.utilities.network_utils:network_utils.py:38 Got a working random port 36625
DEBUG    TriblerConfig:tribler_config.py:111 Get random port 36625 for [libtorrent~anon_listen_port]
INFO     TrackerManager:tracker_manager.py:38 No tracker blacklist file found at /tmp/pytest-of-jenkins/pytest-75/popen-gw5/test_hidden_services0/dot.Tribler/tracker_blacklist.txt.
DEBUG    tribler_core.utilities.network_utils:network_utils.py:38 Got a working random port 26313
DEBUG    TriblerConfig:tribler_config.py:111 Get random port 26313 for [ipv8~port]
DEBUG    UDPEndpoint:endpoint.py:78 Listening at 26313
DEBUG    DiscoveryCommunity:community.py:76 Launching DiscoveryCommunity with prefix b'00027e313685c1912a141279f8248fc8db5899c5df5a'.
WARNING  Notifier:notifier.py:25 Called notification on a non-existing subject NTFY.TRIBLER_STARTED
DEBUG    DiscoveryCommunity:community.py:170 Bootstrapping DiscoveryCommunity, current peers 0
DEBUG    TriblerTunnelCommunity:community.py:76 Launching TriblerTunnelCommunity with prefix b'0002caf481ffae562bcc0f206fd988ec0d246de0dc96'.
INFO     TriblerTunnelCommunity:community.py:140 Exit settings: BT=False, IPv8=False
ERROR    TriblerTunnelCommunity:triblertunnel_community.py:167 Could not retrieve backup exitnode cache, file does not exist!
DEBUG    tribler_core.utilities.network_utils:network_utils.py:38 Got a working random port 6817
DEBUG    tribler_core.utilities.network_utils:network_utils.py:38 Got a working random port 39877
DEBUG    tribler_core.utilities.network_utils:network_utils.py:38 Got a working random port 38879
DEBUG    tribler_core.utilities.network_utils:network_utils.py:38 Got a working random port 18952
DEBUG    tribler_core.utilities.network_utils:network_utils.py:38 Got a working random port 12615
INFO     Session:session.py:388 Session is using state directory: /tmp/pytest-of-jenkins/pytest-75/popen-gw5/test_hidden_services0/dot.Tribler
DEBUG    tribler_core.utilities.network_utils:network_utils.py:38 Got a working random port 44739
DEBUG    TriblerConfig:tribler_config.py:111 Get random port 44739 for [libtorrent~port]
DEBUG    tribler_core.utilities.network_utils:network_utils.py:38 Got a working random port 19590
DEBUG    TriblerConfig:tribler_config.py:111 Get random port 19590 for [libtorrent~anon_listen_port]
INFO     TrackerManager:tracker_manager.py:38 No tracker blacklist file found at /tmp/pytest-of-jenkins/pytest-75/popen-gw5/test_hidden_services0/dot.Tribler/tracker_blacklist.txt.
DEBUG    tribler_core.utilities.network_utils:network_utils.py:38 Got a working random port 41400
DEBUG    TriblerConfig:tribler_config.py:111 Get random port 41400 for [ipv8~port]
DEBUG    UDPEndpoint:endpoint.py:78 Listening at 41400
DEBUG    DiscoveryCommunity:community.py:76 Launching DiscoveryCommunity with prefix b'00027e313685c1912a141279f8248fc8db5899c5df5a'.
WARNING  Notifier:notifier.py:25 Called notification on a non-existing subject NTFY.TRIBLER_STARTED
DEBUG    DiscoveryCommunity:community.py:170 Bootstrapping DiscoveryCommunity, current peers 0
DEBUG    TriblerTunnelCommunity:community.py:76 Launching TriblerTunnelCommunity with prefix b'0002caf481ffae562bcc0f206fd988ec0d246de0dc96'.
INFO     TriblerTunnelCommunity:community.py:140 Exit settings: BT=False, IPv8=False
ERROR    TriblerTunnelCommunity:triblertunnel_community.py:167 Could not retrieve backup exitnode cache, file does not exist!
DEBUG    tribler_core.utilities.network_utils:network_utils.py:38 Got a working random port 35028
DEBUG    tribler_core.utilities.network_utils:network_utils.py:38 Got a working random port 25118
DEBUG    tribler_core.utilities.network_utils:network_utils.py:38 Got a working random port 36126
DEBUG    tribler_core.utilities.network_utils:network_utils.py:38 Got a working random port 11002
DEBUG    tribler_core.utilities.network_utils:network_utils.py:38 Got a working random port 7620
INFO     Session:session.py:388 Session is using state directory: /tmp/pytest-of-jenkins/pytest-75/popen-gw5/test_hidden_services0/dot.Tribler
DEBUG    tribler_core.utilities.network_utils:network_utils.py:38 Got a working random port 10447
DEBUG    TriblerConfig:tribler_config.py:111 Get random port 10447 for [libtorrent~port]
DEBUG    tribler_core.utilities.network_utils:network_utils.py:38 Got a working random port 6676
DEBUG    TriblerConfig:tribler_config.py:111 Get random port 6676 for [libtorrent~anon_listen_port]
INFO     TrackerManager:tracker_manager.py:38 No tracker blacklist file found at /tmp/pytest-of-jenkins/pytest-75/popen-gw5/test_hidden_services0/dot.Tribler/tracker_blacklist.txt.
DEBUG    tribler_core.utilities.network_utils:network_utils.py:38 Got a working random port 18496
DEBUG    TriblerConfig:tribler_config.py:111 Get random port 18496 for [ipv8~port]
DEBUG    UDPEndpoint:endpoint.py:78 Listening at 18496
DEBUG    DiscoveryCommunity:community.py:76 Launching DiscoveryCommunity with prefix b'00027e313685c1912a141279f8248fc8db5899c5df5a'.
WARNING  Notifier:notifier.py:25 Called notification on a non-existing subject NTFY.TRIBLER_STARTED
DEBUG    DiscoveryCommunity:community.py:170 Bootstrapping DiscoveryCommunity, current peers 0
DEBUG    TriblerTunnelCommunity:community.py:76 Launching TriblerTunnelCommunity with prefix b'0002caf481ffae562bcc0f206fd988ec0d246de0dc96'.
INFO     TriblerTunnelCommunity:community.py:140 Exit settings: BT=False, IPv8=False
ERROR    TriblerTunnelCommunity:triblertunnel_community.py:167 Could not retrieve backup exitnode cache, file does not exist!
DEBUG    tribler_core.utilities.network_utils:network_utils.py:38 Got a working random port 19644
DEBUG    tribler_core.utilities.network_utils:network_utils.py:38 Got a working random port 2092
DEBUG    tribler_core.utilities.network_utils:network_utils.py:38 Got a working random port 40624
DEBUG    tribler_core.utilities.network_utils:network_utils.py:38 Got a working random port 45371
DEBUG    tribler_core.utilities.network_utils:network_utils.py:38 Got a working random port 47546
INFO     Session:session.py:388 Session is using state directory: /tmp/pytest-of-jenkins/pytest-75/popen-gw5/test_hidden_services0/dot.Tribler
DEBUG    tribler_core.utilities.network_utils:network_utils.py:38 Got a working random port 15686
DEBUG    TriblerConfig:tribler_config.py:111 Get random port 15686 for [libtorrent~port]
DEBUG    tribler_core.utilities.network_utils:network_utils.py:38 Got a working random port 19783
DEBUG    TriblerConfig:tribler_config.py:111 Get random port 19783 for [libtorrent~anon_listen_port]
INFO     TrackerManager:tracker_manager.py:38 No tracker blacklist file found at /tmp/pytest-of-jenkins/pytest-75/popen-gw5/test_hidden_services0/dot.Tribler/tracker_blacklist.txt.
DEBUG    tribler_core.utilities.network_utils:network_utils.py:38 Got a working random port 6454
DEBUG    TriblerConfig:tribler_config.py:111 Get random port 6454 for [ipv8~port]
DEBUG    UDPEndpoint:endpoint.py:78 Listening at 6454
DEBUG    DiscoveryCommunity:community.py:76 Launching DiscoveryCommunity with prefix b'00027e313685c1912a141279f8248fc8db5899c5df5a'.
WARNING  Notifier:notifier.py:25 Called notification on a non-existing subject NTFY.TRIBLER_STARTED
DEBUG    DiscoveryCommunity:community.py:170 Bootstrapping DiscoveryCommunity, current peers 0
DEBUG    TriblerTunnelCommunity:community.py:76 Launching TriblerTunnelCommunity with prefix b'0002caf481ffae562bcc0f206fd988ec0d246de0dc96'.
INFO     TriblerTunnelCommunity:community.py:140 Exit settings: BT=False, IPv8=False
ERROR    TriblerTunnelCommunity:triblertunnel_community.py:167 Could not retrieve backup exitnode cache, file does not exist!
DEBUG    tribler_core.utilities.network_utils:network_utils.py:38 Got a working random port 19688
DEBUG    tribler_core.utilities.network_utils:network_utils.py:38 Got a working random port 44133
DEBUG    tribler_core.utilities.network_utils:network_utils.py:38 Got a working random port 2787
DEBUG    tribler_core.utilities.network_utils:network_utils.py:38 Got a working random port 26908
DEBUG    tribler_core.utilities.network_utils:network_utils.py:38 Got a working random port 5430
INFO     Session:session.py:388 Session is using state directory: /tmp/pytest-of-jenkins/pytest-75/popen-gw5/test_hidden_services0/dot.Tribler
DEBUG    tribler_core.utilities.network_utils:network_utils.py:38 Got a working random port 59594
DEBUG    TriblerConfig:tribler_config.py:111 Get random port 59594 for [libtorrent~port]
DEBUG    tribler_core.utilities.network_utils:network_utils.py:38 Got a working random port 39661
DEBUG    TriblerConfig:tribler_config.py:111 Get random port 39661 for [libtorrent~anon_listen_port]
INFO     TrackerManager:tracker_manager.py:38 No tracker blacklist file found at /tmp/pytest-of-jenkins/pytest-75/popen-gw5/test_hidden_services0/dot.Tribler/tracker_blacklist.txt.
DEBUG    tribler_core.utilities.network_utils:network_utils.py:38 Got a working random port 43159
DEBUG    TriblerConfig:tribler_config.py:111 Get random port 43159 for [ipv8~port]
DEBUG    UDPEndpoint:endpoint.py:78 Listening at 43159
DEBUG    DiscoveryCommunity:community.py:76 Launching DiscoveryCommunity with prefix b'00027e313685c1912a141279f8248fc8db5899c5df5a'.
WARNING  Notifier:notifier.py:25 Called notification on a non-existing subject NTFY.TRIBLER_STARTED
DEBUG    DiscoveryCommunity:community.py:170 Bootstrapping DiscoveryCommunity, current peers 0
DEBUG    TriblerTunnelCommunity:community.py:76 Launching TriblerTunnelCommunity with prefix b'0002caf481ffae562bcc0f206fd988ec0d246de0dc96'.
INFO     TriblerTunnelCommunity:community.py:140 Exit settings: BT=False, IPv8=False
ERROR    TriblerTunnelCommunity:triblertunnel_community.py:167 Could not retrieve backup exitnode cache, file does not exist!
DEBUG    Download:download.py:289 Saving download config to file /tmp/pytest-of-jenkins/pytest-75/popen-gw5/seeder1/dlcheckpoints/9d5b2dbc52807325bfc28d688f2bb03f8d1e7667.conf
---------------------------- Captured log teardown -----------------------------
INFO     Session:session.py:593 Tribler shutdown state notification:Shutting down IPv8...
WARNING  Notifier:notifier.py:25 Called notification on a non-existing subject NTFY.TRIBLER_SHUTDOWN_STATE
INFO     Socks5Connection:connection.py:204 Closing session, reason stopping
DEBUG    TriblerTunnelCommunity:triblertunnel_community.py:149 Writing exit nodes to cache: /tmp/pytest-of-jenkins/pytest-75/popen-gw5/seeder1/exitnode_cache.dat
DEBUG    Socks5Server:server.py:41 SOCKS5 TCP connection lost
INFO     Socks5Connection:connection.py:204 Closing session, reason unspecified
INFO     Session:session.py:593 Tribler shutdown state notification:Saving configuration...
WARNING  Notifier:notifier.py:25 Called notification on a non-existing subject NTFY.TRIBLER_SHUTDOWN_STATE
INFO     Session:session.py:593 Tribler shutdown state notification:Checkpointing Downloads...
WARNING  Notifier:notifier.py:25 Called notification on a non-existing subject NTFY.TRIBLER_SHUTDOWN_STATE
DEBUG    Download:download.py:588 Stopping b'video.avi'
DEBUG    IPv8Monitor:ipv8_health_monitor.py:74 Mean drift: 0.001830, choke detected: False.
DEBUG    IPv8Monitor:ipv8_health_monitor.py:82 Current walk_interval: 0.500000.
DEBUG    Download:download.py:289 Saving download config to file /tmp/pytest-of-jenkins/pytest-75/popen-gw5/seeder1/dlcheckpoints/9d5b2dbc52807325bfc28d688f2bb03f8d1e7667.conf
INFO     Session:session.py:593 Tribler shutdown state notification:Shutting down Downloads...
WARNING  Notifier:notifier.py:25 Called notification on a non-existing subject NTFY.TRIBLER_SHUTDOWN_STATE
INFO     Session:session.py:593 Tribler shutdown state notification:Shutting down Libtorrent Manager...
WARNING  Notifier:notifier.py:25 Called notification on a non-existing subject NTFY.TRIBLER_SHUTDOWN_STATE
INFO     Session:session.py:593 Tribler shutdown state notification:Shutting down IPv8...
WARNING  Notifier:notifier.py:25 Called notification on a non-existing subject NTFY.TRIBLER_SHUTDOWN_STATE
DEBUG    TriblerTunnelCommunity:triblertunnel_community.py:149 Writing exit nodes to cache: /tmp/pytest-of-jenkins/pytest-75/popen-gw5/test_hidden_services0/dot.Tribler/exitnode_cache.dat
INFO     Session:session.py:593 Tribler shutdown state notification:Saving configuration...
WARNING  Notifier:notifier.py:25 Called notification on a non-existing subject NTFY.TRIBLER_SHUTDOWN_STATE
INFO     Session:session.py:593 Tribler shutdown state notification:Shutting down Libtorrent Manager...
WARNING  Notifier:notifier.py:25 Called notification on a non-existing subject NTFY.TRIBLER_SHUTDOWN_STATE
INFO     Session:session.py:593 Tribler shutdown state notification:Shutting down IPv8...
WARNING  Notifier:notifier.py:25 Called notification on a non-existing subject NTFY.TRIBLER_SHUTDOWN_STATE
DEBUG    TriblerTunnelCommunity:triblertunnel_community.py:149 Writing exit nodes to cache: /tmp/pytest-of-jenkins/pytest-75/popen-gw5/test_hidden_services0/dot.Tribler/exitnode_cache.dat
INFO     Session:session.py:593 Tribler shutdown state notification:Saving configuration...
WARNING  Notifier:notifier.py:25 Called notification on a non-existing subject NTFY.TRIBLER_SHUTDOWN_STATE
INFO     Session:session.py:593 Tribler shutdown state notification:Shutting down IPv8...
WARNING  Notifier:notifier.py:25 Called notification on a non-existing subject NTFY.TRIBLER_SHUTDOWN_STATE
DEBUG    TriblerTunnelCommunity:triblertunnel_community.py:149 Writing exit nodes to cache: /tmp/pytest-of-jenkins/pytest-75/popen-gw5/test_hidden_services0/dot.Tribler/exitnode_cache.dat
INFO     Session:session.py:593 Tribler shutdown state notification:Saving configuration...
WARNING  Notifier:notifier.py:25 Called notification on a non-existing subject NTFY.TRIBLER_SHUTDOWN_STATE
INFO     Session:session.py:593 Tribler shutdown state notification:Shutting down IPv8...
WARNING  Notifier:notifier.py:25 Called notification on a non-existing subject NTFY.TRIBLER_SHUTDOWN_STATE
DEBUG    TriblerTunnelCommunity:triblertunnel_community.py:149 Writing exit nodes to cache: /tmp/pytest-of-jenkins/pytest-75/popen-gw5/test_hidden_services0/dot.Tribler/exitnode_cache.dat
INFO     Session:session.py:593 Tribler shutdown state notification:Saving configuration...
WARNING  Notifier:notifier.py:25 Called notification on a non-existing subject NTFY.TRIBLER_SHUTDOWN_STATE
INFO     Session:session.py:593 Tribler shutdown state notification:Shutting down IPv8...
WARNING  Notifier:notifier.py:25 Called notification on a non-existing subject NTFY.TRIBLER_SHUTDOWN_STATE
DEBUG    TriblerTunnelCommunity:triblertunnel_community.py:149 Writing exit nodes to cache: /tmp/pytest-of-jenkins/pytest-75/popen-gw5/test_hidden_services0/dot.Tribler/exitnode_cache.dat
INFO     Session:session.py:593 Tribler shutdown state notification:Saving configuration...
WARNING  Notifier:notifier.py:25 Called notification on a non-existing subject NTFY.TRIBLER_SHUTDOWN_STATE
INFO     Session:session.py:593 Tribler shutdown state notification:Shutting down IPv8...
WARNING  Notifier:notifier.py:25 Called notification on a non-existing subject NTFY.TRIBLER_SHUTDOWN_STATE
DEBUG    TriblerTunnelCommunity:triblertunnel_community.py:149 Writing exit nodes to cache: /tmp/pytest-of-jenkins/pytest-75/popen-gw5/test_hidden_services0/dot.Tribler/exitnode_cache.dat
INFO     Session:session.py:593 Tribler shutdown state notification:Saving configuration...
WARNING  Notifier:notifier.py:25 Called notification on a non-existing subject NTFY.TRIBLER_SHUTDOWN_STATE
```

--- a/src/tribler-core/tribler_core/modules/tunnel/test_full_session/test_tunnel_community.py
+++ b/src/tribler-core/tribler_core/modules/tunnel/test_full_session/test_tunnel_community.py
@@ -217,7 +217,7 @@ async def test_anon_download(enable_ipv8, proxy_factory, session, video_seeder_s
 
 
 @pytest.mark.asyncio
-@pytest.mark.timeout(30)
+@pytest.mark.timeout(40)
 @pytest.mark.tunneltest
 async def test_hidden_services(enable_ipv8, proxy_factory, session, hidden_seeder_session, video_tdef):
     """

